### ISSUE TITLE
White Rectangular Spaces At The Bottom & The Middle of The Screen

### ISSUE THREAD

**Issue by ghost** [brave/brave-browser#17838]
## Description 
While opening a new website from a new tab (or sometimes with the currently opened tab also), I see some awkward rectangular white spaces, occurring at the bottom of the windows and sometimes even in the middle. 50 per cent of the time, while opening a website, these gets showed. These blanks are not specific to a certain website and are random! They were not present before the 1.29.76 update!

## Steps to Reproduce

   1.  Open 10-15 tabs
   2.  Type a search term or enter URL's (all of different websites)
   3. Chances are you'll also get these blanks!

## Actual result:
White rectangular spaces gets showed up!

## Expected result:
White rectangular spaces shouldn't showed!

## Reproduces how often: 
Intermittent issue

## Brave version (brave://version info)

1.29.76

- Can you reproduce this issue with the current release? Y

## Screenshots or Videos:

https://user-images.githubusercontent.com/80682093/131846784-3a57a64d-e0c2-447d-9ced-fb2d3813b1d7.mp4

You can see I tested for it in 6 websites, and the blanks were shown in 3 of them.

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No

**Comment by rebron** [comment]
@Nihal247 I only see the one rectangular image on the New Tab Page. What were the other three rectangles? You've got a a few extensions too, I'm not sure if any of those are causing those rectangles to show up.

**Comment by Nihal247** [comment]
Yes these rectangular images mostly gets showed on the New Tab Page. Those other three rectangles were the same too. Sometimes, even a big rectangular box also gets displayed, as you can see in 00:37 of the video. And yeas, I disabled all the extensions and it didn't changed anything. These boxes weren't displayed before v1.29.76!

**Comment by rebron** [comment]
<img width="868" alt="Screen Shot 2021-09-21 at 10 17 08 AM" src="https://user-images.githubusercontent.com/5951041/134217289-941cd614-bacb-4ca6-9b9c-eb14571e938e.png">

**Comment by Nihal247** [comment]
I turned it off, but I'm still seeing it.
